docs/examples/bicepsettings.json

```diff
@@ -0,0 +1,41 @@
+{
+  "Linter": {

```

Comment:
>Linter [](http://example.com/codeflow?start=3&length=6)

[MVP question] Should we use a shorter name for this file? Since this will be used for more than just linter rules, why not just `bicep.json`?

Reply:
We should discuss during triage.

Some examples I've seen:

webpack.config.js
NuGet.Config
tsconfig.json (typescript)
settings.json (vscode settings, in a .vscode folder)
package.json
tslint.json
launch.json (in a .vscode folder)
.editorconfig
.eslintrc.*

I'd suggest one of:
bicep.json
bicepconfig.json

Reply:
Yup sounds good. Alex and I were just chatting on teams and also came up with `bicep.config`.

Reply:
Let's try to close on this during Monday's meeting when I'm out.

Reply:
decision: bicepconfig.json

Reply:
John did this.